ibu-imager/seedcreator/seedcreator.go

```diff
@@ -234,15 +234,10 @@ func (s *SeedCreator) createContainerList(ctx context.Context) error {
 		return err
 	}
 	images := strings.Split(output, "\n")
-	catalogImages, err := s.getCatalogImages(ctx)
+	images, err = s.filterCatalogImages(ctx, images)
 	if err != nil {
 		return err
 	}
-	s.log.Infof("Removing list of catalog images from full image list, catalog images to remove %s", catalogImages)
```

Comment:
So wait, the old code was also filtering catalog images? So what difference does your new code make?

Reply:
https://github.com/openshift-kni/lifecycle-agent/pull/220/files#diff-af44b48262b65ed56f5f9d016b4adb3fa8f72e0313e666d5a325b97426b718a8R501